### ISSUE TITLE
set(JSON_LIB_PATH Thirdparty/jsoncpp/dist)

### ISSUE THREAD

**Issue by Stephanie-ustc** [ZiyunGe1999/SimpleMVS#1]
Hi，Thank you for sharing.
CMakeLists.txt has the following statement: set(JSON_LIB_PATH Thirdparty/jsoncpp/dist), but there is no thirdparty / jsoncpp / dist in the given folder.

**Comment by Stephanie-ustc** [comment]
Expect bloggers to update

**Comment by ZiyunGe1999** [comment]
> Hi，Thank you for sharing.
> CMakeLists.txt has the following statement: set(JSON_LIB_PATH Thirdparty/jsoncpp/dist), but there is no thirdparty / jsoncpp / dist in the given folder.

This project is still in development! Sorry I haven't uploaded the thirdparty folder. Anyway, thank you for your issue.

**Comment by ZiyunGe1999** [comment]
> Expect bloggers to update

Yeah, I plan to write a detailed blog about this project, which may be a good example for people new to MVS. I will finish this ASAP.

**Comment by blc1996** [comment]
@ZiyunGe1999 Looking forward to you update!

**Comment by LHXhh** [comment]
@ZiyunGe1999 Hi，would you have any bloggers about this project, thank you very much.
